linux-armv6/crosstool-ng.config

```diff
@@ -602,22 +610,21 @@ CT_CC_GCC_LIBMUDFLAP=y
 CT_CC_GCC_LIBGOMP=y
 CT_CC_GCC_LIBSSP=m
 CT_CC_GCC_LIBQUADMATH=y
-CT_CC_GCC_LIBSANITIZER=y
+# CT_CC_GCC_LIBSANITIZER=y
```

Comment:
disable sanitizer as there is a gcc bug https://gcc.gnu.org/bugzilla/show_bug.cgi?id=100379 with new linux kernel and not being fixed until later version